aten/src/ATen/native/quantized/cpu/qconcat.cpp

```diff
@@ -0,0 +1,50 @@
+#include <ATen/ATen.h>
+#include <ATen/core/Type.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
+#include <ATen/quantized/Quantizer.h>
+
+#include <algorithm>
+#include <vector>
+
+namespace at { namespace native {
+namespace {
+template <bool ReLUFused = false>
+class QCat final : public c10::OperatorKernel {
+ public:
+  Tensor operator()(const std::vector<Tensor>& qxs, long axis,
```

Comment:
is this the interface we agreed on? are we going to merge it with the existing "cat_" op by passing in a dummy Tensor with output scale and output zero_point?

Reply:
This is the same signature that the `nn.cat` has + scale + zero_point. What is the alternative?

Reply:
BTW, the scale and zero_point are optional